excel-importer/src/main/java/uk/gov/hmcts/ccd/definition/store/excel/parser/ParseContext.java

```diff
@@ -153,6 +159,31 @@ public ParseContext registerStateForCaseType(String caseTypeId, StateEntity stat
         return this;
     }
 
+    public CategoryEntity getCategory(String caseTypeId, String categoryId) {
+        final Map<String, CategoryEntity> categories = categoryByCaseTypes.get(caseTypeId);
+
+        if (null == categories) {
+            return null;
+        }
+
+        return categories.get(categoryId);
+    }
+
+    public ParseContext registerCaseTypeForCategory(String caseTypeId, CategoryEntity categoryEntity) {
+        categoryByCaseTypes.computeIfAbsent(caseTypeId, k -> Maps.newHashMap());
+
+        final Map<String, CategoryEntity> categoryStates = categoryByCaseTypes.get(caseTypeId);
+
+        if (categoryStates.containsKey(categoryEntity.getCategoryId())) {
+            throw new SpreadsheetParsingException("Category already registered for ID: "
+                + categoryEntity.getCategoryId());
+        }
+
+        categoryStates.put(categoryEntity.getCategoryId(), categoryEntity);
+
+        return this;
```

Comment:
Obviously don't need this if void method

Reply:
done but will check